extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -140,16 +126,16 @@ export class NetCoreTool extends ShellExecutionHelper {
 	}
 
 	/**
-	 * This function checks if the installed dotnet version is between minSupportedNetCoreVersion (inclusive) and maxSupportedNetCoreVersionCutoff (exclusive).
-	 * When maxSupportedNetCoreVersionCutoff is not set, the latest dotnet version is assumed to be supported and only the min version is checked.
+	 * This function checks if the installed dotnet version is atleast minSupportedNetCoreVersion.
+	 * Versions lower than minSupportedNetCoreVersion aren't supported for building projects.
 	 * Returns: True if installed dotnet version is supported, false otherwise.
 	 * 			Undefined if dotnet isn't installed.
 	 */
 	private async isNetCoreVersionSupported(): Promise<boolean | undefined> {
```

Comment:
Maybe we should rename this `isNetCoreVersionSupportedForBuild` or something since that's the only scenario that this really applies too AFAIK.

Honestly what I'd even more prefer is that build has its own special check it does before running the dotnet command instead of sticking it in the general run command function...that already bit us once and I can easily see it happening again.

Reply:
opened https://github.com/microsoft/azuredatastudio/issues/17742 to track moving out the build specific checks